Apps/Sandcastle/gallery/Imagery Layers Split.html

```diff
@@ -68,27 +68,41 @@
 var slider = document.getElementById('slider');
 viewer.scene.imagerySplitPosition = (slider.offsetLeft) / slider.parentElement.offsetWidth;
 
-var dragStartX = 0;
+var handler = new Cesium.ScreenSpaceEventHandler(slider);
 
-document.getElementById('slider').addEventListener('mousedown', mouseDown, false);
-window.addEventListener('mouseup', mouseUp, false);
+var bMoveActive = false;
 
-function mouseUp() {
-  window.removeEventListener('mousemove', sliderMove, true);
-}
+function move(movement){
```

Comment:
Formatting should be consistant with the rest of the code base.
`function move(movement){` -> `function move(movement) {`
`if(!bMoveActive){` -> `if (!bMoveActive) {`